cmd/svcat/broker/get_cmd.go

```diff
@@ -17,22 +17,27 @@ limitations under the License.
 package broker
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
 	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/output"
 	"github.com/kubernetes-incubator/service-catalog/pkg/svcat/service-catalog"
 	"github.com/spf13/cobra"
 )
 
-type getCmd struct {
+// GetCmd contains the information needed to get a broker or list of brokers
+type GetCmd struct {
```

Comment:
exposed for testing?

Reply:
Yeah, and in general I think these things should be public - they're the public facing part of the package.

Reply:
👍